src/models/forms/neuburgForm.js

```diff
@@ -1,10 +1,19 @@
 // @flow
 
+const VERSION = 1.0
```

Comment:
Schould we make this just a number maybe? Android uses a versionCode for this which is just a incrementing counter

Reply:
Yeah we can do that. This would probably make it easier, you're right :)

Reply:
So `"1"` instead of `1.0`, right?

Reply:
Yeah.. probably a number would be easier 🤔